department-of-veterans-affairs/caseflow
# Queue | Use disposition ID

### Description
One of the reasons the bug described in https://github.com/department-of-veterans-affairs/appeals-deployment/pull/1480 went unnoticed is because issue dispositions were not round-trippable. In multiple places on the FE, we were transforming disposition descriptions to fit the context, resulting in a mismatch between what the BE expected and what the FE provided. This obviates the description formatting problem by passing the disposition ID.

To ensure backwards compatibility, this adds a `disposition_id` attribute to Issues, which is passed to Queue as `disposition` within `WorkQueue::IssueSerializer`.

### Acceptance Criteria 
- [ ] Setting Issue disposition during attorney check out persists when viewing issue later

### Testing Plan
1. Go to attorney Draft Decision checkout, set dispositions for issues
2. Complete check out, go to appeal detail view as the assigned judge
3. Confirm dispositions render correctly in IssueListItem
